SSA/Experimental/Bits/Fast/BitStream.lean

```diff
@@ -380,18 +380,18 @@ local infix:20 " ≈ʷ " => EqualUpTo w
 @[simp] theorem ofBitVec_add : ofBitVec (x + y) ≈ʷ (ofBitVec x) + (ofBitVec y)  := by
   sorry
 
-@[simp] theorem ofBitVec_neg : ofBitVec (- x) ≈ʷ  - (ofBitVec x) := by
-  sorry
-
+@[refl]
 theorem equal_up_to_refl : a ≈ʷ a := by
-  intros  j _
+  intros j _
   rfl
 
+@[symm]
 theorem equal_up_to_symm (e : a ≈ʷ b) : b ≈ʷ a := by
   intros j h
   symm
   exact e j h
 
+@[trans]
```

Comment:
i would like a [Trans](https://leanprover-community.github.io/mathlib4_docs/Init/Prelude.html#Trans) instance here

Reply:
Also can be more general with (a = b up to m) and (b = c up to w) then (a = c up to min m w)